test/e2e/framework.go

```diff
@@ -2728,20 +2718,36 @@ func (data *TestData) gracefulExitFlowAggregator(covDir string) error {
 // collectAntctlCovFiles collects coverage files for the antctl binary from the Pod and saves them to the coverage directory
 func (data *TestData) collectAntctlCovFiles(podName string, containerName string, nsName string, covDir string) error {
```

Comment:
I suppose you want this file to be generic function to collect different coverage files?  
You may need to rename this function if so, and update the comment of this function.